tests/test_validate_item_collection.py

```diff
@@ -0,0 +1,665 @@
+"""
+Description: Test the validator
+
+"""
+__authors__ = "James Banting", "Jonathan Healy"
```

Comment:
```suggestion
__authors__ = ["James Banting", "Jonathan Healy"]
```

Reply:
These were in all the test files. I just erased them all.